.github/ISSUE_TEMPLATE/bug-report.yml

```diff
@@ -0,0 +1,58 @@
+name: 🐞 Bug
+description: Found a bug? Use this template
+title: "Bug: Write a small description here"
+labels:
+  [
+    "status: todo ⏳",
+    "🐞 bug",
+    "hacktoberfest"
+  ]
```

Comment:
I previously said to not just copy paste. We donot have a label called `status: todo ⏳` or `🐞 bug`
Please fix these kind of things across the whole code. Use the current labels that we have, for example `bug`